### PR TITLE
[dgram] Fix garbage dgram output

### PR DESCRIPTION
**Pull request by jprestwo**
Fixes #947

 - Removed QEMU specific option
 - Added 6lowpan support for A101

Signed-off-by: James Prestwood <james.prestwood@intel.com>

### CODE REVIEW

**Comment by pfalcon** [comment]
Good thoughts come to different people at the same time ;-). Feel free to pick up frdm_k64f setting from https://github.com/01org/zephyr.js/pull/961

**Comment by jprestwo** [comment]
Added config options for K64 from @pfalcon's #961

File: src/Makefile.base [hunk 1]
```diff
@@ -17,10 +17,12 @@ ccflags-y += -I$(JERRY_BASE)/jerry-core
 ccflags-y += -I$(ZEPHYR_BASE)/drivers
 ccflags-y += -I$(ZJS_BASE)/outdir/include
 
+ifeq ($(ZJS_DGRAM), y)
 ifeq ($(BOARD), arduino_101)
 obj-y += ../deps/zephyr/samples/bluetooth/gatt/ipss.o
```
Comment (grgustaf) [inline src/Makefile.base]: See whether it works to enable OCF and dgram at the same time, since both include this.

**Comment by grgustaf** [comment]
+1

**Pull request merged**